tools/ncf.py

```diff
@@ -550,6 +550,7 @@ def generate_technique_content(technique, methods):
   for method_call in technique["method_calls"]:
     method_name = method_call["method_name"]
     method_info = methods[method_name]
+    # regex to match quote characters not preceded by a backslash
```

Comment:
it seems to me it does a little more than backslash, or else the regexes lines 581-583 are not complete

Reply:
No it's exactly that, the regex here in 554 is using a negative lookbehind: 

> (?<!text)

to match quotes not preceded by a backslash

then 581-583 use the regexp defined in 554 to replace the unescaped quotes by an escaped quote

Reply:
My bad ! Thank you for the clarification